docs/CONTRIBUTING.rst

```diff
@@ -26,13 +26,18 @@ will serve the community better if other folks get involved.
 
 There are lots of ways to contribute -- it's not all about code!
 
-* Ask questions on Github using the `issue tracker <https://github.com/catalyst-cooperative/pudl/issues>`__.
+* If you need help, someone else might need it too - ask for help in `Github
+  Discussions
+  <https://github.com/orgs/catalyst-cooperative/discussions/categories/help-me>`__
+  and maybe the ensuing discussion will be useful to other people too!
 * `Suggest new data and features <https://github.com/catalyst-cooperative/pudl/issues/new?template=feature_request.md>`__ that would be useful.
```

Comment:
Maybe we should remove the feature request issue template? I think starting with a discussion and then maybe creating an issue is likely a better pattern.

Reply:
Yeah, that makes sense - you might like this: https://github.com/catalyst-cooperative/.github/pull/1